### DISCUSSION TITLE
Orthography of Japanobotrychum

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#16] [Q&A]
Is Masamune's original spelling _Japanobotrychum_ (see protolog at [https://dl.lib.ntu.edu.tw/s/tj/item/824567](https://dl.lib.ntu.edu.tw/s/tj/item/824567); so also his label on the holotype, to be found at [https://tai2.ntu.edu.tw/](https://tai2.ntu.edu.tw/)) considered corrigible to _Japanobotrychium_ under Art. 60.1? This is the course that has often been taken, including in PPG I, but I am curious to know whether anyone has ever sought an opinion on it, and what spelling Masamune used in Short Flora of Formosa (1936) when he recombined _J. arisanense_ into _Botrychium_.

**Comment by msundue** [comment]
I have not seen the Short Flora of Forma yet, but I see that Carl Christensen reprinted the name as originally published in the 3rd supplement of his Index Filicum (1934). I also see that in the Index Pteridophytarum Japonicarum Tagawa (1959) revised the spelling to what is in use now. I could see both sides on this one, but ICBN Art. 60 Example 14 provides guidance:

Ex. 14. Cacalia "napeaefolia" and Senecio "napeaefolius" are to be cited as Cacalia napaeifolia DC. (1838) and Senecio napaeifolius (DC.) Sch. Bip. (1845), respectively; the specific epithet refers to the resemblance of the leaves to those of the genus Napaea L. (not "Napea")...

From that example I would go with Japanobotrychium
